src/tools/dev/scripts/bv_support/bv_vtkh.sh

```diff
@@ -158,6 +170,7 @@ function build_vtkh
     vopts="${vopts} -DVTKM_DIR=${VISITDIR}/vtkm/${VTKM_VERSION}/${VISITARCH}"
     vopts="${vopts} -DENABLE_MPI=OFF"
     vopts="${vopts} -DENABLE_OPENMP=OFF"
+    vopts="${vopts} -DBUILD_SHARED_LIBS:BOOL=OFF"
```

Comment:
Change `-DBUILD_SHARED_LIBS:BOOL=OFF` to `-DBUILD_SHARED_LIBS:BOOL=ON`. VTK-h should be built shared, VTK-m is built static.